src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -614,8 +614,8 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter)
     if (f.exists) {
       addedClasspath = ClassPath.join(addedClasspath, f.path)
       intp.addUrlsToClassPath(f.toURI.toURL)
-      echo("Added '%s' to classpath.".format(f.path, intp.global.classPath.asClassPathString))
-      repldbg("Added '%s'.  Your new classpath is:\n\"%s\"".format(f.path, intp.global.classPath.asClassPathString))
+      echo("Added '%s' to classpath.".format(f.path))
+      repldbg("Added '%s'. Your new classpath is:\n\"%s\"".format(f.path, intp.global.classPath.asClassPathString))
```

Comment:
@adriaanm is refactoring/rewriting. Also, it would be natural to change this to interpolated string, which is immune to the error. There should be a style rule about format strings containing only `%s`.